troposphere/static/js/components/modals/instance/InstanceLaunchWizardModal.react.js

```diff
@@ -25,14 +25,12 @@ export default React.createClass({
     getInitialState: function() {
 
         //===========================
-        // BasicOptions variables
-        //===========================
-
         // We might have these
         let image = this.props.image ? this.props.image : null;
         let project = this.props.project ? this.props.project : null;
+        let provider = null;
         let view = image ? "BASIC_VIEW" : "IMAGE_VIEW";
-        let instanceName = image ? image.attributes.name : null;
+        let imageName = image ? image.get("name") : null;
```

Comment:
:+1: this is a good catch. We should **not** be using `attributes`. Thanks Connor.